nbqa/__main__.py

```diff
@@ -303,19 +305,19 @@ def _preserve_config_files(nbqa_config: Optional[str], tmpdirname: str) -> None:
     tmpdirname
         Temporary directory to store converted notebooks in.
     """
-    if nbqa_config is None:
-        return
-    Path(tmpdirname).joinpath(
-        Path(nbqa_config).resolve().relative_to(Path.cwd())
-    ).parent.mkdir(parents=True, exist_ok=True)
-    shutil.copy(
-        str(nbqa_config),
-        str(
-            Path(tmpdirname).joinpath(
-                Path(nbqa_config).resolve().relative_to(Path.cwd())
-            )
-        ),
-    )
+    if nbqa_config is not None:
+        config_files = [nbqa_config]
```

Comment:
Here we expect the config name to be the default ones looked up by the tool right? For instance, if we are using flake8, then we set `config = .flake8`. But if we name it to something like `.flake8_nb` and give it to nbqa, it won't work, even though we copy the file to the tmp directory. I don't see the config parameter being passed as a command line parameter to the CLI command. flake8 has a command line parameter `--config` to specify a custom configuration file.

```Python
    output = subprocess.run(
        ["python", "-m", command, str(arg), *kwargs],
        stderr=subprocess.PIPE,
        stdout=subprocess.PIPE,
        cwd=tmpdirname,
        env=env,
    )
```

I know we can use addopts to specify `--config .flake8_nb`. But from a user's perspective the user would expect it to work by just setting the `--nbqa-config` setting right?Should it be considered a bug?

Reply:
@girip11 that's a really good point, thanks...I think we should just document that this needs to be one of the tool's standard configuration files and that users should consult the tools' docs to find out how to name/use them